assets/main.js

```diff
@@ -0,0 +1,149 @@
+window.onload = function() {
+  function toPlugSlider(
+    nameSlider,
+    nameSliderWrapper,
+    isSliderBlog,
+    isArrows,
+    isPagination,
+    namePaginationWrapper
+  ) {
+    const slider = document.getElementById(nameSlider);
+    const slideWrapper = document.getElementById(nameSliderWrapper);
+    const slides = slideWrapper.children;
+    let wrapperDots = slider.querySelector(namePaginationWrapper);
+    let arrows;
+    let arrowRight;
+    let arrowLeft;
+    let dots;
+
+    function renderArrows() {
+      let arrows = document.createElement("div");
+      let arrowLeft = document.createElement("a");
+      let arrowRight = document.createElement("a");
+
+      arrows.className = "arrows";
+      arrowLeft.className = "arrows__item arrows__item_left";
+      arrowRight.className = "arrows__item arrows__item_right";
+
+      arrows.appendChild(arrowLeft);
+      arrows.appendChild(arrowRight);
+      return arrows;
+      console.log(arrows);
+    }
+
+    function addArrows() {
+      if (isSliderBlog) {
+        if (slides.length > 3) {
+          slider.appendChild(renderArrows());
+        }
+      } else {
+        if (slides.length > 1) {
+          slider.appendChild(renderArrows());
+        }
+      }
+    }
+
+    function renderDot(valueDot) {
+      const dotWrapper = document.createElement("div");
+      const dot = document.createElement("label");
+      const dotInput = document.createElement("input");
+
+      wrapperDots.className = "pagination";
+      dotWrapper.className = "pagination__item";
+      dot.setAttribute("for", `slide_${valueDot}`);
+      dotInput.type = "radio";
+      dotInput.id = `slide_${valueDot}`;
+      dotInput.name = "slide";
+      dotInput.value = `slide_${valueDot}`;
+      valueDot === 0 && (dotInput.checked = true);
```

Comment:
It's a bad practice to use ternary operator instead of `if` statement for a functionality that makes some side-effects.
Use ternary operator only for variables value assignment.